dist_q_learning/main.py

```diff
@@ -353,21 +388,16 @@ def selected_mentor(state, kwargs=None):
     if args.agent == "pess_gln":
         agent_kwargs.update({"quantile_i": args.quantile})
 
-    lr = args.learning_rate if args.learning_rate is not None else (
-        1. if str(args.trans) == "2" else 0.1)  # 1. if deterministic
-
     if args.n_steps > 0:
         agent = agent_init(
             num_actions=env.num_actions,
             env=env,
-            gamma=0.99,
+            gamma=0.95,
```

Comment:
I think it helped at one point - perhaps allows immediate reward to pump up rewards faster, because cartpole doesn't need to see *too* far into the future, only far enough to see if this trajectory makes it fall over